pkg/controller/servicemesh/controlplane/controller_test.go

```diff
@@ -102,9 +94,8 @@ func TestMerge(t *testing.T) {
 }
 
 func TestCyclicTemplate(t *testing.T) {
-	reconcileControlPlane := newTestReconciler(nil)
+	reconcileControlPlane := newTestReconciler()
```

Comment:
nit, but does it make sense to rename this to reconciledControlPlane? reconcile is a verb.

Reply:
I agree. But in addition to this variable, the whole type name is also wrong. It's been bothering me for a while, but I didn't want to change it, as it's been like this from the start. I'll rename these things in a different PR.